02.Git.Local/02.GIT.Local.md

```diff
@@ -0,0 +1,55 @@
+  337  git clone git@github.com:DAlekseichik/sa.it-academy.by.git
```

Comment:
Could you please use here git markup (just put into code section and add some header-description